### PR TITLE
Servlet

### PR DESCRIPTION
**Pull request by psygrammator**

### CODE REVIEW

File: Servlet/src/main/webapp/WEB-INF/web.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE web-app PUBLIC
```
Comment (kartius) [inline Servlet/src/main/webapp/WEB-INF/web.xml:1]: You can remove web.xml

File: Servlet/src/main/java/top/jsoft/service/GetService.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+import top.jsoft.model.Product;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by psygrammator
+ * group jsoft.top
+ */
+public class GetService {
```
Comment (kartius) [inline Servlet/src/main/java/top/jsoft/service/GetService.java:14]: Please rename to OrderService

**Review by kartius — APPROVED** [review]

File: Servlet/src/main/java/top/jsoft/service/GetService.java (hunk 2 above)
Comment (psygrammator) [reply Servlet/src/main/java/top/jsoft/service/GetService.java]: Done, i understand that it is better to create your own service for each servlet.

File: Servlet/src/main/webapp/WEB-INF/web.xml (hunk 1 above)
Comment (psygrammator) [reply Servlet/src/main/webapp/WEB-INF/web.xml]: done, ty

**Pull request merged**